github/repos_contents.go

```diff
@@ -106,6 +121,46 @@ func (s *RepositoriesService) GetReadme(ctx context.Context, owner, repo string,
 	return readme, resp, nil
 }
 
+// GetReadme gets the Readme file for the repository.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/contents/#get-the-readme
+func (s *RepositoriesService) GetReadmeCustom(ctx context.Context, owner, repo string, opt *RepositoryContentGetOptions) (string, *Response, error) {
```

Comment:
👍 for `GetReadmeCustom` instead of having different methods like `GetReadmeRaw`, `GetReadmeHTML`, etc. I read your comment about this suggestion on the related issue.